NewSpring/ApollosProject
# (DEV-3697) Private Prayer Requests are able to be returned to that user.

## DESCRIPTION
We needed to update the api call to ignore the `isPublic` flag and ignore it when creating a prayer.

### What does this PR do, or why is it needed?
This PR removes references to the `IsPublic` attribute and automatically sets it to True no matter what when creating from the app.

### How do I test this PR?
1. Login on the Graphql playground
2. Confirm which prayers are set to "IsPublic" in Alpha-Rock
3. Run this mutation query to create a new prayer:
```
mutation {
  addPublicPrayerRequest(FirstName: "Test", LastName: "Bro", Text: "Jesus Rocks", CampusId: 16, CategoryId: 1, IsAnonymous: "True") {
    id
    firstName
    lastName
    text
    requestedByPersonAliasId
    campusId
    categoryId
    flagCount
    prayerCount
    isAnonymous
    person {
      id
      firstName
      lastName
    }
  }
}
```
4. Confirm the newly created mutation has `IsPublic` set to true in Alpha-Rock.
5. Get all user prayer requests running this query:
```
query {
  getCurrentPersonPrayerRequests {
    id
    firstName
    lastName
    text
    enteredDateTime
  }
}
```
6. Confirm all user prayers are listed.
---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3697](https://newspring.atlassian.net/browse/DEV-3697)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
